harness/determined/cli/user.py

```diff
@@ -41,65 +41,70 @@
 )
 
 
-@login_sdk_client
 def list_users(args: Namespace) -> None:
-    resp = bindings.get_GetMaster(setup_session(args))
-    users_list = client.list_users(active=None if args.all else True)
+    sess = cli.setup_session(args)
+    d = client.Determined._from_session(sess)
+    resp = bindings.get_GetMaster(sess)
+    users_list = d.list_users(active=None if args.all else True)
     renderer = FullUser  # type: Any
     if resp.to_json().get("rbacEnabled"):
         renderer = FullUserNoAdmin
     render.render_objects(renderer, users_list)
 
 
-@login_sdk_client
-def activate_user(parsed_args: Namespace) -> None:
-    user_obj = client.get_user_by_name(parsed_args.username)
+def activate_user(args: Namespace) -> None:
+    sess = cli.setup_session(args)
+    d = client.Determined._from_session(sess)
+    user_obj = d.get_user_by_name(args.username)
     user_obj.activate()
 
 
-@login_sdk_client
-def deactivate_user(parsed_args: Namespace) -> None:
-    user_obj = client.get_user_by_name(parsed_args.username)
+def deactivate_user(args: Namespace) -> None:
+    sess = cli.setup_session(args)
+    d = client.Determined._from_session(sess)
+    user_obj = d.get_user_by_name(args.username)
     user_obj.deactivate()
 
 
-def log_in_user(parsed_args: Namespace) -> None:
-    if parsed_args.username is None:
+def log_in_user(args: Namespace) -> None:
+    if args.username is None:
         username = input("Username: ")
     else:
-        username = parsed_args.username
+        username = args.username
 
     message = "Password for user '{}': ".format(username)
     password = getpass.getpass(message)
 
-    token_store = authentication.TokenStore(parsed_args.master)
-    token = authentication.do_login(parsed_args.master, username, password, certs.cli_cert)
-    token_store.set_token(username, token)
-    token_store.set_active(username)
+    token_store = authentication.TokenStore(args.master)
+    utp = authentication.login(args.master, username, password, cli.cert)
```

Comment:
Not your bug (exists both before this PR and after), but I just noticed that if you aren't logged in and you try to log in with a bad password you get

```
Failed to log in user: Unauthenticated: Please use 'det user login <username>' for password login, or for Enterprise users logging in with an SSO provider, use 'det auth login --provider=<provider>'.
```

Just in case you're interested in fixing it here.

Reply:
I am not interested in fixing it here, sorry.